src/hooks/eip4626/mutations/useDeployVault.ts

```diff
@@ -43,12 +51,18 @@ export function useDeployVault() {
         walletInterface as WalletInterface,
         {
           args: [salt, vaultDetails, feeConfig],
-          value: ethers.parseUnits(DEPLOY_VALUE, 18),
+          value: ethers.parseUnits(currentDeployValueParsed, 18),
         },
       );
 
       //@TODO clear query cache for the list of vaults
       return deployResult;
     },
   });
+
+  useEffect(() => {
+    setIsTxError(mut.isError);
+  }, [mut.isError]);
```

Comment:
why have separate field for the error, can we use the error from mutation directly?